tests/bluetooth/controller/common/src/helper_pdu.c

```diff
@@ -155,6 +155,20 @@ void helper_pdu_encode_start_enc_rsp(struct pdu_data *pdu, void *param)
 	pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_START_ENC_RSP;
 }
 
+void helper_pdu_encode_pause_enc_req(struct pdu_data *pdu, void *param)
+{
+	pdu->ll_id = PDU_DATA_LLID_CTRL;
+	pdu->len = offsetof(struct pdu_data_llctrl, pause_enc_req) + sizeof(struct pdu_data_llctrl_pause_enc_req);
```

Comment:
It is a nitpick... due to line length it will not go through checkpatch. It will end with warning, but for some time now warning also make builds to fail. Wouldn't it be better to make the code to follow coding rules where it can?

Reply:
I am not too sure what to do here, we have already many places where line length exceeds 100 chars. Maybe it's better with a separate pull request where we fix all linelength violations in one go?

Reply:
Yes, that is a good idea to fix that in separate PR.

Reply:
At some point this needs fixing; until now we haven't focused too much on this. Maybe best is to create one PR where we fix all linelength issues in one sweep?
There's also some typos a.o. in ull_llcp_enc: recieved -> received

Reply:
I my humble opinion it is better to do it sooner than later. Then we will avoid unnecessary rework later.